src/Globe/TileMesh.js

```diff
@@ -46,7 +48,9 @@ function TileMesh(params, builder, geometryCache) {
     this.geometry = defaultValue(geometryCache, new TileGeometry(params, builder));
     this.normal = params.center.clone().normalize();
 
-    this.distance = params.center.length();
+    var worldNormal = new THREE.Vector3(0, 0, 1).applyQuaternion(this.OBB().getWorldQuaternion());
```

Comment:
See commit message:
> Previous computation assumed we were using a globe.
The new one produces the same result for a globe, but also
work in other situations (planar mode for instance).